src/ol/format/ogc/filter.js

```diff
@@ -219,15 +219,15 @@ ol.format.ogc.filter.like = function(propertyName, pattern,
  */
 ol.format.ogc.filter.Filter = function(tagName) {
 
-  goog.base(this);
+  ol.Object.call(this);
 
   /**
    * @private
    * @type {!string}
    */
   this.tagName_ = tagName;
 };
-goog.inherits(ol.format.ogc.filter.Filter, ol.Object);
+ol.inherits(ol.format.ogc.filter.Filter, ol.Object);
```

Comment:
Not introduced here, but this stands out as odd to me (see https://github.com/openlayers/ol3/pull/5252/files#r65902356).